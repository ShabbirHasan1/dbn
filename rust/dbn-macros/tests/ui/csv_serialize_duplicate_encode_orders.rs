use dbn_macros::CsvSerialize;

#[derive(CsvSerialize)]
#[repr(C)]
struct Record {
    #[dbn(encode_order(1))]
    pub a: u8,
    pub b: u8,
    #[dbn(encode_order(1))]
    pub c: u8,
}

fn main() {}
