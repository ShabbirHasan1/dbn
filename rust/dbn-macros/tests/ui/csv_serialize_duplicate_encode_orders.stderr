error: Specified duplicate encode order `1` for field
  --> tests/ui/csv_serialize_duplicate_encode_orders.rs:9:5
   |
9  | /     #[dbn(encode_order(1))]
10 | |     pub c: u8,
   | |_____________^
